### PR TITLE
docs(release-notes): updates for 1.4 release

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

Updated the known issues
Added improvements to Sample Web UI

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -27,6 +24,11 @@ The toolkit enables robust scaling through:
 #### MPS
 - **AllowList:** Allowlist enables administrators to specify which devices (GUIDs) can connect to MPS. Removing "developer mode" in a previous version broke Allowlist functionality. In this release, MPS AllowList functionality allows devices to connect if they are listed in the MPS database and present the proper credentials. If a device either is not listed in the MPS database or has the wrong credentials, MPS will refuse the connection. Along with this change, we have improved the MPS Devices API to make it very easy to add, edit, and remove devices from the MPS database. MPS API documentation can be found out on [Swagger](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0#/)
 
+#### Sample Web UI
+- **Improvements:** We have made some minor changes and improvements in the Sample Web UI to stream line profile creation and provide additional information on the devices page.
+   * When setting a new random password in the CIRA Config and Profiles the UI now sets the password length for the user, removing the need for the password length field.  The REST APIs still support user defined values from 8 to 32 for password length.
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:29]: It was hard to tell what/who is doing the action in that bullet item. I may be getting it wrong, but...

 * The UI now sets the password length for new random passwords, removing the need for the password length field in the CIRA Config and Profiles.  The REST APIs still support user defined values from 8 to 32 for password length.****

File: docs/release-notes.md [hunk 2]
```diff
@@ -37,4 +39,7 @@ The toolkit enables robust scaling through:
 #### UI-Toolkit
 - **KVM freeze intermittently:** We have added a small delay in handling mouse interactions that prevents us from flooding the AMT channel.  There are still a few occasions where KVM could still freeze.  We are still root causing this new issue
 #### RPS
-- **Failure to add Device:** If an RPS call to add new device information and credentials to Vault fails, the RPS should revert the changes to the other service and emit an error message. Currently RPS will provide an error, but it does not revert the changes. Resolving the failing issue with the service and then re-running the activation restores the data integrity between the database and Vault.
\ No newline at end of file
+- **[Data shouldn't be added if vault calls fail](https://github.com/open-amt-cloud-toolkit/rps/issues/254):** Currently, if vault calls fail (i.e. during creation of domain), the domain is successfully added to the database despite not being added to vault. Ideally, the database entry should be rolled back if vault call fails.
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:42]: Currently, if vault calls fail (e.g., during the creation of domain), the domain is successfully added to the database despite not being added to vault. Ideally, the database entry should be rolled back if vault call fails.

**Review by kimberlyxdavis — DISMISSED** [review]

File: docs/release-notes.md [hunk 3]
```diff
@@ -7,15 +7,12 @@ This section outlines key features changes between versions 1.3 and 1.4 for Open
...
+   *  Cloud provider agnostic [deployment documentation](https://open-amt-cloud-toolkit.github.io/docs/1.4/Kubernetes/kubernetes/).
+   *  Improved routing of KVM and SOL sessions when multiple instances of MPS are deployed.
+   *  An increase in the number of CIRA connections that can be handled per MPS instance. Detailed guidance will be provided with the LTS release.
 
 #### MPS Router
 - **Reverse Proxy:** In this release we are adding a new required component to the toolkit deployment, the MPS Router.  This small containerized GO serivce acts as a reverse proxy between the API Gateway and MPS.  When Intel&reg; AMT devices make a CIRA connection with MPS, the MPS registers the device GUID and its MPS instance ID with the MPS Router.  All MPS API calls coming from the API Gateway are first sent to the MPS Router before being forwarded to the correct MPS instance.  As with all components of the Open AMT Cloud Toolkit, this service is open source and is located in the [MPS Router](https://github.com/open-amt-cloud-toolkit/mps-router) repository.
```
Comment (rsdmike) [inline docs/release-notes.md:18]: This small containerized GO serivce acts ->  This small containerized Golang-based service acts

File: docs/release-notes.md [hunk 4]
```diff
@@ -27,6 +24,11 @@ The toolkit enables robust scaling through:
...
 
+#### Sample Web UI
+- **Improvements:** We have made some minor changes and improvements in the Sample Web UI to stream line profile creation and provide additional information on the devices page.
+   * When setting a new random password in the CIRA Config and Profiles the UI now sets the password length for the user, removing the need for the password length field.  The REST APIs still support user defined values from 8 to 32 for password length.
+   * On the devices page, we have added the device Name, GUID, and tags to help with easily finding this information for each device.
```
Comment (rsdmike) [inline docs/release-notes.md:30]: why is Name capitalized? i think not needed.

File: docs/release-notes.md [hunk 5]
```diff
@@ -7,15 +7,12 @@ This section outlines key features changes between versions 1.3 and 1.4 for Open
 
 ### Additions
 #### RPS & MPS
```
Comment (rsdmike) [inline docs/release-notes.md:9]: Breaking changes:

-Authentication for Websockets -- that was listed as a known issue last release, that has been resolved this release. JWT required for KVM/SOL connections
- Environment variables removed in MPS
- Vault KeyName changes in RPS (removed profile prefix from key name)
- MQTT!!!!
- Global Credentials removed for MPS
- No default passwords

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: docs/release-notes.md (hunk 1 above)
Comment (matt-primrose) [reply docs/release-notes.md:29]: Updated

File: docs/release-notes.md (hunk 2 above)
Comment (matt-primrose) [reply docs/release-notes.md:42]: Done

File: docs/release-notes.md (hunk 3 above)
Comment (matt-primrose) [reply docs/release-notes.md:18]: Done

File: docs/release-notes.md (hunk 4 above)
Comment (matt-primrose) [reply docs/release-notes.md:30]: English is hard...

File: docs/release-notes.md (hunk 5 above)
Comment (matt-primrose) [reply docs/release-notes.md:25]: done

File: docs/release-notes.md [hunk 6]
```diff
@@ -2,30 +2,52 @@
...
+
 ## Key Feature Changes for 1.4
-This section outlines key features changes between versions 1.3 and 1.4 for Open AMT Cloud Toolkit. 
+This section outlines key features changes between versions 1.3 and 1.4 for Open AMT Cloud Toolkit.
+
+### Breaking Changes
+#### MPS & RPS: 
+   - **WebSocket Authentication Updated:** Kong Authenication for Websockets is now required for KVM and SOL connections.  The same JWT that is used for REST calls is also used for KVM and SOL connections.
```
Comment (rsdmike) [inline docs/release-notes.md:15]: can also might be better. Ideally its different and very short lived, i.e. like 1 min

File: docs/release-notes.md [hunk 7]
```diff
@@ -2,30 +2,52 @@
...
+- **Reverse Proxy:** In this release we are adding a new required component to the toolkit deployment, the MPS Router.  This small containerized Golang-based serivce acts as a reverse proxy between the API Gateway and MPS.  When Intel&reg; AMT devices make a CIRA connection with MPS, the MPS registers the device GUID and its MPS instance ID with the MPS Router.  All MPS API calls coming from the API Gateway are first sent to the MPS Router before being forwarded to the correct MPS instance.  As with all components of the Open AMT Cloud Toolkit, this service is open source and is located in the [MPS Router](https://github.com/open-amt-cloud-toolkit/mps-router) repository.
```
Comment (rsdmike) [inline docs/release-notes.md:34]: serivce => service

File: docs/release-notes.md [hunk 8]
```diff
@@ -2,30 +2,52 @@
...
+- **Simplified API:** The MPS API received some additional attention in this release. In 1.2 we added the ability to add metadata to the MPS database for tracking device hostname and tags. In the API, we had this as a separate “metadata” endpoint. In 1.4 we merged the “metadata” endpoint into the “devices” endpoint to greatly simplify the MPS API organization structure. The “devices” endpoint supports GET, POST, and PATCH for managing all device information, including metadata. You can find updated API documentation on SwaggerHub for both [RPS](https://app.swaggerhub.com/apis-docs/rbheopenamt/rps/1.4.0) and [MPS](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0).
```
Comment (rsdmike) [inline docs/release-notes.md:45]: This is a breaking change

File: docs/release-notes.md [hunk 9]
```diff
@@ -2,30 +2,52 @@
...
+- **Simplified API:** The MPS API received some additional attention in this release. In 1.2 we added the ability to add metadata to the MPS database for tracking device hostname and tags. In the API, we had this as a separate “metadata” endpoint. In 1.4 we merged the “metadata” endpoint into the “devices” endpoint to greatly simplify the MPS API organization structure. The “devices” endpoint supports GET, POST, and PATCH for managing all device information, including metadata. You can find updated API documentation on SwaggerHub for both [RPS](https://app.swaggerhub.com/apis-docs/rbheopenamt/rps/1.4.0) and [MPS](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0). 
+
+#### Sample Web UI
+- **Improvements:** We have made some minor changes and improvements in the Sample Web UI to stream line profile creation and provide additional information on the devices page.
```
Comment (rsdmike) [inline docs/release-notes.md:48]: streamline is one word

**Review by rsdmike — CHANGES_REQUESTED** [review]

**Review by rsdmike — APPROVED** [review]

**Pull request merged**